.travis.yml

```diff
@@ -86,15 +130,15 @@ script:
      fi
    # Print the compiler version
    - $MYCXX -v
-   # Build the code and do a quick check (debug mode) or a full tests run (non-debug mode)
-   - if [ $DEBUG == "NO" ]; then
-        export MFEM_TEST_TARGET="test";
-     else
-        export MFEM_TEST_TARGET="check";
-     fi
+        
    # Build and check/test MFEM, its examples and miniapps
```

Comment:
Added option in MFEM build system to specify number of mpi ranks to use for parallel problems.  Code coverage has issues with more than one rank, as each process will try to output to the same process cov file ( known issue ).  Running with one MPI rank sufficient for coverage purposes.